src/gui/georeferencing_dialog.cpp

```diff
@@ -385,6 +400,13 @@ void GeoreferencingDialog::declinationChanged()
 	setValueIfChanged(declination_edit, georef->getDeclination());
 }
 
+// slot
+void GeoreferencingDialog::gridScaleFactorChanged()
+{
+	const QSignalBlocker block(scale_factor_edit);
+	scale_factor_edit->setValue(georef->getGridScaleFactor());
+}
```

Comment:
If we should remove signal `Georeferencing::gridScaleFactorChanged()`, this code would go to `transformationChanged`.

Reply:
Yes.

Reply:
Just found that the code is already there. Well, the (comined) grid scale factor is feature which came before this PR ;-)